python/nav/web/templates/machinetracker/mac_search.html

```diff
@@ -75,6 +75,13 @@
                   {{ form.netbios }}
                 </label>
               </div>
+
+              <div class="small-6 column">
+                <label for="{{ form.vendor.auto_id }}" title="{{ form.vendor.help_text }}">
+                  {{ form.vendor.label }}
+                  {{ form.vendor }}
+                </label>
+              </div>
```

Comment:
Something doesn't look quite right: The checbox ends up on a new row, but right-aligned?


![image](https://github.com/user-attachments/assets/c80659fe-bf3d-4af8-8c00-2515a794ba46)